src/Analyzers/Core/Analyzers/RemoveUnusedParametersAndValues/AbstractRemoveUnusedParametersAndValuesDiagnosticAnalyzer.SymbolStartAnalyzer.BlockAnalyzer.cs

```diff
@@ -130,69 +124,6 @@ bool HasConditionalDirectives()
 
                         return false;
                     }
-
-                    static bool IsSingleThrowNotImplementedOperation(IOperation firstBlock)
```

Comment:
this helper moved to an extension method as we want to use it in "initialize property"